source/contracts/reporting/FeeWindow.sol

```diff
@@ -108,7 +105,7 @@ contract FeeWindow is DelegationTarget, VariableSupplyToken, Extractable, Initia
         }
 
         if (_totalFeeStake == 0) {
-            return;
+            return true;
```

Comment:
😡 It makes me so angry that Solidity compiler didn't error on this previously... good catch.